app/build.gradle

```diff
@@ -19,6 +19,7 @@ android {
         versionCode major * 10000 + minor * 100 + patch
         versionName "${major}.${minor}.${patch}"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables.useSupportLibrary = true
```

Comment:
Possibly no need for that as we are targeting quite new Android version and vector drawables are used all over the app.